modules/ui_extra_networks.py

```diff
@@ -397,20 +442,8 @@ def toggle_visibility(is_visible):
 
     state_visible = gr.State(value=False)
     button.click(fn=toggle_visibility, inputs=[state_visible], outputs=[state_visible, container, button])
-
-    def refresh(request: gr.Request):
-        res = []
-
-        for pg in ui.stored_extra_pages:
-            pg.refresh(request)

```

Comment:
This is the only place where the models are being reloaded. You need to add `pg.refresh(request)` back at somewhere in the updates.